examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
```

Comment:
Yeah. You're right. Forgot about creating new proto each time the passed object prototype is not the system default one (`Object.getPrototypeOf(obj) === Object.prototype`)